### ISSUE TITLE
No visibility options or template options when trying to generate request

### ISSUE THREAD

**Issue by stein-afp** [OfficeDev/microsoft-teams-apps-requestateam#270]
Hi

I've deployed request a team and I got the Request-a-team app added to teams. I can see (and click) both the "From scratch" and "Use template" buttons, however I'm stuck once I hit one of them. I'm being taken to the next page, whereas for the From Scratch button will ask what visibility the team should have, and the Use template button will as what template to use. Both of these dropdown boxes are totally empty. I don't know where the visibility options comes from, but I can see the template list in sharepoint (in the Teams template page/collection in sharepoint).

What can be wrong?

Also when I first enter the request-a-team app - I'm getting the following error message: Office365Groups.ListGroupMembers failed: { "error": { "code": "Request_ResourceNotFound", "message": "Resource 'xxxxxxxxxx-a3d1-4eaf-8c7d-7593d6e35c33' does not exist or one of its queried reference-property objects are not present.", "innerError": { "date": "2021-06-11T12:45:14", "request-id": "46a44dfb-9602-4813-xxxx-xxxxxxxxxxxxxx", "client-request-id": "46a44dfb-9602-4813-xxxx-xxxxxxxxxxxxx" } } }

Thanks
Stein

**Comment by hyde311** [comment]
I am having the same issue.  Any suggestions?

**Comment by imgibason** [comment]
Hi @stein-afp,

Did you ever get this resolved.  This is my first time running through this deployment and I am getting the same thing.  When I click From scratch or Use template the drop down menus are emtpy.

I was also getting the Office365Groups.ListGroupMembers failed error however I resolved this by making sure I put in the correct groupid I had saved during the deployment guide.

In the Configuring the Power App section of the deployment guide on step 8 make sure you have the correct groupid listed and saved.

The groupid comes from Step 2: Create Admin Group section.

Regards!
Ian...
